practices/p4/task2.js

```diff
@@ -0,0 +1,97 @@
+const Shipyard = function () {
+    this.isSameType = (ship, yard) => {
+        return ship === yard
+    }
```

Comment:
Раз метод становится публичным, тип верфи можно инкапсулировать внутри класса, чтобы не было необходимости его передавать:
```suggestion
    this.isSameType = (type) => {
        return type === this.type;
    }
```

Reply:
Справедливо. Исправил. Спасибо